<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultsHistoryEntryWidget</class>
 <widget class="QWidget" name="ResultsHistoryEntryWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>90</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>90</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>90</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_7">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QCheckBox" name="selection_box_">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0,0,1,0,0">
       <property name="spacing">
        <number>20</number>
       </property>
       <item>
        <widget class="QLabel" name="ep_label_">
         <property name="text">
          <string>EPName DeviceType)</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="date_icon_label_">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../app_resources.qrc">:/icons/resources/icons/calendar_icon.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="date_label_">
           <property name="text">
            <string>MMM dd, yyyy</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="time_icon_label_">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../app_resources.qrc">:/icons/resources/icons/time_icon.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="time_label_">
           <property name="text">
            <string>00:00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="error_icon_label_">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../app_resources.qrc">:/icons/resources/icons/ep_info_icon.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ElidedLabel" name="error_label_" native="true"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="TTFT_icon_label_">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../app_resources.qrc">:/icons/resources/icons/edu_icon.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="TTFT_title_label_">
           <property name="text">
            <string>TTFT:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="TTFT_label_">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>8</number>
         </property>
         <item>
          <widget class="QLabel" name="TPS_icon_label_">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../../app_resources.qrc">:/icons/resources/icons/summarize_icon.png</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="TPS_title_label_">
           <property name="text">
            <string>TPS:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="TPS_label_">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,0,1">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="cpu_bullet_">
           <property name="minimumSize">
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cpu_title_label_">
           <property name="text">
            <string>CPU</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="cpu_label_">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,0,1">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="ram_bullet_">
           <property name="minimumSize">
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ram_title_label_">
           <property name="text">
            <string>RAM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="ram_label_">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="0,0,1">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="gpu_bullet_">
           <property name="minimumSize">
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="gpu_title_label_">
           <property name="text">
            <string>GPU</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="gpu_label_">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="0,0,1">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="gpu_vram_bullet_">
           <property name="minimumSize">
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="gpu_vram_title_label_">
           <property name="text">
            <string>GPU VRAM</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="gpu_vram_label_">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="0,0,1">
         <property name="spacing">
          <number>4</number>
         </property>
         <item>
          <widget class="QLabel" name="os_bullet_">
           <property name="minimumSize">
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>8</width>
             <height>8</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="os_title_label_">
           <property name="text">
            <string>OS</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="os_label_">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="open_button_">
     <property name="minimumSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>36</width>
       <height>36</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../app_resources.qrc">
       <normaloff>:/icons/resources/icons/expand_left.png</normaloff>:/icons/resources/icons/expand_left.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ElidedLabel</class>
   <extends>QWidget</extends>
   <header>ui/widgets/elided_label.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../app_resources.qrc"/>
 </resources>
 <connections/>
</ui>
